nengo_loihi/axons.py

```diff
@@ -97,3 +99,31 @@ def validate(self):
                 for atom, axon_id in zip(self.cx_atoms, axon_ids):
                     n_populations = self.target.axon_populations(axon_id)
                     assert 0 <= atom < n_populations
+
+
+class AxonGroup(object):
```

Comment:
Could you add docstrings for AxonGroup/SynapseGroup/ProbeGroup as well?

Reply:
I've added short ones. If you can think of anything more to add, let me know.